load/InputFileset.py

```diff
@@ -385,12 +346,14 @@ def batchName(self):
 
 
 if (__name__ == '__main__'):
+	from InputProcessor import *	
+
 	config = Config()
 	s3Client = AWSSession.shared().s3Client
```

Comment:
I think this line can be removed. The same for the lines 353 and 354

Reply:
I get an error if I remove it